x/oracle/abci_test.go

```diff
@@ -0,0 +1,227 @@
+package oracle_test
+
+import (
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/medibloc/panacea-core/v2/types/testsuite"
+	"github.com/medibloc/panacea-core/v2/x/oracle"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/stretchr/testify/suite"
+	"testing"
+	"time"
+)
+
+type abciTestSuite struct {
+	testsuite.TestSuite
+
+	uniqueID string
+
+	oraclePubKey  cryptotypes.PubKey
+	oracleAddr    sdk.AccAddress
+	oraclePubKey2 cryptotypes.PubKey
+	oracleAddr2   sdk.AccAddress
+	oraclePubKey3 cryptotypes.PubKey
+	oracleAddr3   sdk.AccAddress
+
+	newOraclePubKey cryptotypes.PubKey
+	newOracleAddr   sdk.AccAddress
```

Comment:
I put these variables in a struct instead of global.
Because if a variable is declared globally, it can be accessed by other test files in the same package.
Separated for test independence.